### PR TITLE
adds the ts-34 double barrel shotgun, replacing the one marines get in the req vendor ((NOT PFC))

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds the TS-34 double barrel shotgun. Basically a double barrel that can't burst fire (lol) and has a few more attachment options and new sprite.

## Why It's Good For The Game
dear god why did marines get this by default. Better sprite anyway.

## Changelog
:cl:
add: Added the ts-34 sprite
balance: added the ts-34, a double barrel with no burst fire, and a few more attachment options
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -26. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Chaznoodles** [comment]
Put it in the Ops vendor instead of the Marine ones if you want them to be ((NOT PFC)), because as it stands you've put 5 per vendor and ship maps tend to have a few of those vendors.

File: code/modules/projectiles/guns/shotguns.dm [hunk 1]
```diff
@@ -430,6 +430,41 @@ can cause issues with ammo types getting mixed up during the burst.
 	recoil = 3
 	recoil_unwielded = 5
 
+//-------------------------------------------------------
+//MARINE DOUBLE SHOTTY
+
+/obj/item/weapon/gun/shotgun/double/marine
+	name = "TS-34 double barrel shotgun"
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/shotguns.dm:437]:
```suggestion
	name = "\improper TS-34 double barrel shotgun"
```

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-29). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**